os/services/rpl-border-router/Makefile.rpl-border-router

```diff
@@ -0,0 +1,10 @@
+CFLAGS += -DBUILD_WITH_RPL_BORDER_ROUTER=1
+
+include $(CONTIKI)/Makefile.identify-target
+
+# Include 'embedded' module for non-native platforms
+ifneq ($(TARGET),native)
+	MODULES += os/services/rpl-border-router/embedded
+endif
+# Include optional target-specific module
+MODULES += os/services/rpl-border-router/$(TARGET)
```

Comment:
I am not very convinced about this one I'm afraid. We are going through a lot of effort to abstract various parts of `os` including peripheral HALs and all that. Are we sure we want to open this precedent of platform-specific code under `os`? Perhaps we can come up with a clean way of pulling this from within `arch/platform`?

Reply:
Are you bothered only with the `sky` folder here or also `native`?

Reply:
OK now I see what you've done. Hmmm, yeah native is platform-dependent too isn't it. Not sure, I'll sleep over it! :D

Reply:
Yep no worries, nothing urgent, see this really as an initial proposal. I would really want the BR moved form `examples` to `os`, but still quite open how we actually do this. The sky code is so little code that I believe it would be easy to move out. Native is really different, as this is embedded BR vs NBR.